e2e_tests/standard/test_cli.py

```diff
@@ -55,6 +56,10 @@ def test_sync_container(self, container_type, environment):
                 container_id = container._id
                 container_sys_id = container._sys_id
 
+                self.stop_synchronization_process(container)
```

Comment:
Should we call `container.sync()` before?

Reply:
`container.wait()` is called in 55 line